src/libs/API.js

```diff
@@ -62,6 +62,26 @@ function isAuthTokenRequired(command) {
     ], command);
 }
 
+/**
+ * Determines if we're supposed to retry the given request
+ * @param {Object} request
+ * @returns {Boolean}
+ */
+function shouldRetry(request) {
+    return lodashGet(request, 'data.shouldRetry', false);
+}
+
+/**
+ * Add the request back to the queue to be retried
+ * It will either get retried now, or later when we're back online
+ * @param {Object} request
+ */
+function retry(request) {
```

Comment:
The function name `retry` is a bit odd considering the only thing that infers that its for retrying is the context in which its called. The code itself is just a basic post, right?

Reply:
it uses an existing request and re-posts it. also it does hook to the `then` and `catch` of the original

But yeah it's primarily to reuse the logic and spare some duplicate post code

Reply:
If it's only about the variable name I thought of naming it `retryRequest` I don't know why I didn't go with that...

Reply:
I'll push a change to rename this